perf/perf_duplicate_probe.py

```diff
@@ -43,18 +43,19 @@ def setUp(self):
         for package in deps:
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
+        self.fail_flag = False
+
+    def _check_duplicate_probe(self, output):
+        if 'select_task_rq_fair' in output and 'select_task_rq_fair_' in output:
+            self.fail_flag = True
 
     def test_probe(self):
         output = process.run("perf probe select_task_rq_fair:0", sudo=True)
         output = output.stderr.decode("utf-8")
-        fail_flag = False
-        if 'select_task_rq_fair' in output and 'select_task_rq_fair_' in output:
-            fail_flag = True
+        self._check_duplicate_probe(output)
         output = genio.read_all_lines("/sys/kernel/debug/tracing/kprobe_events")
```

Comment:
Can you paste a sample output of this file so that it is easier to understand the change?

Reply:
PASS scenario:
```
# perf probe select_task_rq_fair:0
Added new event:
  probe:select_task_rq_fair (on select_task_rq_fair)

You can now use it in all perf tools, such as:

	perf record -e probe:select_task_rq_fair -aR sleep 1

# cat /sys/kernel/debug/tracing/kprobe_events
p:probe/select_task_rq_fair _text+1883064
```

FAIL scenario:
```
[root@ltcden14-lp8 ~]# perf probe select_task_rq_fair:0
Added new event:
  probe:select_task_rq_fair (on select_task_rq_fair)
  probe:select_task_rq_fair_1 (on select_task_rq_fair)
You can now use it in all perf tools, such as:

	perf record -e probe:select_task_rq_fair -aR sleep 1

[root@ltcden14-lp8 ~]# cat /sys/kernel/debug/tracing/kprobe_events
p:probe/select_task_rq_fair _text+1883064
p:probe/select_task_rq_fair_1 _text+1883064
```